File: server/当前版本/server/zlj1/game_server/src/com/imop/lj/gameserver/mall/msg/GCNextQueueCd.java
package com.imop.lj.gameserver.mall.msg;

import com.imop.lj.core.msg.MessageType;
import com.imop.lj.gameserver.common.msg.GCMessage;
/**
 * 返回下个限时列表上架CD
 *
 * @author CodeGenerator, don't modify this file please.
 */
public class GCNextQueueCd extends GCMessage{
	
	/** cd */
	private long cd;

	public GCNextQueueCd (){
	}
	
	public GCNextQueueCd (
			long cd ){
			this.cd = cd;
	}

	@Override
	protected boolean readImpl() {

	// cd
	long _cd = readLong();
	//end



		this.cd = _cd;
		return true;
	}
	
	@Override
	protected boolean writeImpl() {

	// cd
	writeLong(cd);


		return true;
	}
	
	@Override
	public short getType() {
		return MessageType.GC_NEXT_QUEUE_CD;
	}
	
	@Override
	public String getTypeName() {
		return "GC_NEXT_QUEUE_CD";
	}

	public long getCd(){
		return cd;
	}
		
	public void setCd(long cd){
		this.cd = cd;
	}
}

File: u3u2/Assets/Scripts/app/net/s2c/GCNextQueueCd.cs

using System;
namespace app.net
{
/**
 * 返回下个限时列表上架CD
 *
 * @author CodeGenerator, don't modify this file please.
 */
public class GCNextQueueCd :BaseMessage
{
	/** cd */
	private long cd;

	public GCNextQueueCd ()
	{
	}

	protected override void ReadImpl()
	{
	// cd
	long _cd = ReadLong();


		this.cd = _cd;
	}
	
	protected override void WriteImpl()
    {
        return;
    }
	
	public override short GetMessageType() 
	{
		return (short)MessageType.GC_NEXT_QUEUE_CD;
	}
	
	public override string getEventType()
	{
		return MallGCHandler.GCNextQueueCdEvent;
	}
	

	public long getCd(){
		return cd;
	}
		

}
}